templates/common/_base/units/openvswitch.service

```diff
@@ -0,0 +1,2 @@
+name: "openvswitch.service"
+enabled: {{if eq .NetworkType "OVNKubernetes"}}true{{else if eq .NetworkType "OpenShiftSDN"}}true{{else}}false{{end}}
```

Comment:
I think you can drop this and instead just have:
```
After=openvswitch.service
Requires=openvswitch.service
```
in `ovs-configuration.service` - then it will pull it in as a dependency only if that service itself is enabled.

Reply:
that's part of @runcom patch not mine, it was already merged